docusaurus/src/pages/index.js

```diff
@@ -37,39 +43,86 @@ export default function Home() {
+                        <h2 style={{display:"flex", paddingRight: "10px", margin: "0px"}}>OpenZiti</h2>
+                        <img src={"/img/github.svg"} style={{display:"flex", alignItems:"center", height: "75px"}}/>
+                    </div>
+                    <div className={styles.whozitibullets}>
+                        <div style={{display:"flex", flexDirection: "column"}}>
+                            <p>Host everything yourself</p>
+                            <p>Some people learn best by <Highlight>doing</Highlight></p>
+                            <p>Fully configurable - you're running it all after all!</p>
+                        </div>
+                        <div className={styles.indexCtas}>
+                            <Link className="button button--primary" to="#build-a-network">
+                                <p>Get OpenZiti</p>
+                            </Link>
+                        </div>
+                    </div>
+                </div>
+                <div className={styles.whoziticontainer}>
+                    <div className={styles.whozitititlelogo}>
+                        <h2 style={{display:"flex", paddingRight: "10px", margin: "0px"}}>ZEDS</h2>
+                        <img src={"/img/zeds.png"} style={{display:"flex", alignItems:"center", height: "75px"}}/>
+                    </div>
+                    <div className={styles.whozitibullets}>
+                        <div style={{display:"flex", flexDirection: "column"}}>
+                            <p><Highlight>No installation necessary</Highlight>! Just sign up and go!</p>
+                            <p>Easiest path for SDK-only solutions</p>
+                            <p>No network customization</p>
+                        </div>
+                        <div style={{display:"flex", alignItems:"center"}}>
+                            <div className={styles.indexCtas}>
+                                <Link className="button button--primary" to="https://zeds.openziti.org/">
+                                    <p>Get ZEDS!</p>
+                                </Link>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div className={styles.whoziticontainer}>
+                    <div className={styles.whozitititlelogo}>
+                        <h2 style={{display:"flex", paddingRight: "10px", margin: "0px"}}>CloudZiti</h2>
+                        <img src={"/img/nf.svg"} style={{display:"flex", alignItems:"center", width: "75px"}}/>
+                    </div>
+                    <div className={styles.whozitibullets}>
+                        <div style={{display:"flex", flexDirection: "column"}}>
+                            <p>Easy to get started, fully customizable</p>
+                            <p><Highlight>Managed and supported</Highlight> by NetFoundry</p>
+                            <p>OpenZiti with none of the maintenance</p>
```

Comment:
"NetFoundry OpenZiti" sounds awkward to me. I think we should say "Managed and supported by NetFoundry" or "Fully-managed CloudZiti".